Projekty/Projekt2/Gruszkowska_Gryszkiewicz_Chylak/KM1.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Szkoda, że nie opisano jak działa funkcja. Warto wyjaśnić zastosowany algorytm i tym samym pokazać włożony wysiłek.  Zwłaszcza, że zastosowanie chmury słów jest wyróżnikiem tej grupy, więc warto to uwypuklić. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>